docker-compose.nightly-functional-tests.yml

```diff
@@ -3,10 +3,10 @@ version: '2.1'
 services:
     citizen-integration-tests:
       image: hmcts/cmc-citizen-integration-tests:${CITIZEN_INTEGRATION_TESTS_VERSION:-latest}
-      command: test:integration --grep '^(?!.*@nightly)'
+      command: test:integration --grep @nightly
       environment:
         - WEB_DRIVER_HOST=remote-webdriver
-        - CITIZEN_APP_URL=${TEST_URL}
+        - CITIZEN_APP_URL=${TEST_URL}ok,
```

Comment:
`ok` :)

Reply:
haha that must be what i wanted to reply to you in slack :)